File: TOMIMSV4/Entity/Sistema_Jornada/clsBeVW_Fecha_Recepcion_TMS_Ticket.vb
Public Class clsBeVW_Fecha_Recepcion_TMS_Ticket
    Implements ICloneable

    Public Property IdTicket() As Integer = 0
    Public Property IdOrdenCompraEnc() As Integer = 0
    Public Property IdRecepcionEnc() As Integer = 0
    Public Property Fecha_Ingreso() As Date = Date.Now
    Public Property Fecha_Creacion() As Date = Date.Now
    Public Property Fecha_Recepcion() As Date = Date.Now

    Sub New()
        '#EJC: Add your constructor here... 
    End Sub

    Public Function Clone() As Object Implements System.ICloneable.Clone
        Return MyBase.MemberwiseClone()
    End Function

End Class

File: WMS.EntityCore/Ticket/clsBeVW_Fecha_Recepcion_TMS_Ticket.cs
namespace WMS.EntityCore.Ticket
{
    public class clsBeVW_Fecha_Recepcion_TMS_Ticket : ICloneable
    {
        public int IdTicket { get; set; } = 0;
        public int IdOrdenCompraEnc { get; set; } = 0;
        public int IdRecepcionEnc { get; set; } = 0;
        public DateTime Fecha_Ingreso { get; set; } = DateTime.Now;
        public DateTime Fecha_Creacion { get; set; } = DateTime.Now;
        public DateTime Fecha_Recepcion { get; set; } = DateTime.Now;

        public clsBeVW_Fecha_Recepcion_TMS_Ticket()
        {
            // EJC: Add your constructor here...
        }

        public object Clone()
        {
            return this.MemberwiseClone();
        }
    }
}
